ppr-ui/src/components/search/SearchBarList.vue

```diff
@@ -0,0 +1,194 @@
+                <v-btn icon small style="pointer-events: all;">
+                  <v-icon v-if="displayGroup[item.group]" class="expand-icon" color="primary">mdi-chevron-up</v-icon>
+                  <v-icon v-else class="expand-icon" color="primary">mdi-chevron-down</v-icon>
+                </v-btn>
+              </v-col>
+            </v-row>
+          </v-list-item-content>
+        </template>
+        <template v-else class="search-list">
+          <v-list-item
+            v-if="displayGroup[item.group]"
+            class="copy-normal"
+            @click="selectSearchType(item)"
+          >
+            <v-list-item-title>
+              {{ item.searchTypeUI }}
+            </v-list-item-title>
+          </v-list-item>
+        </template>
+      </template>
+  </v-select>
+</template>
+<script lang="ts">
+import { defineComponent, reactive, toRefs, computed, ref } from '@vue/composition-api'
+import { useGetters } from 'vuex-composition-helpers'
+import { MHRSearchTypes, SearchTypes } from '@/resources'
+import { UISearchTypes, APISearchTypes } from '@/enums'
+import { SearchTypeIF } from '@/interfaces' // eslint-disable-line no-unused-vars
+import { getFeatureFlag } from '@/utils'
+
+export default defineComponent({
+  name: 'SearchBarList',
+  emits: ['selected'],
+  props: {
+    defaultSelectedSearchType: {
+      type: Object as () => SearchTypeIF
+    },
+    defaultCategoryMessage: {
+      type: String,
+      default: ''
+    }
+  },
+  setup (props, { emit }) {
+    const {
+      isRoleStaffReg,
+      hasPprRole,
+      hasMhrRole
+    } = useGetters<any>([
+      'isRoleStaffReg',
+      'hasPprRole',
+      'hasMhrRole'
+    ])
+    const searchSelect = ref(null)
+    const localState = reactive({
+      searchTypes: UISearchTypes,
+      searchTypeValues: APISearchTypes,
+      selectedSearchType: props.defaultSelectedSearchType,
+      categoryMessage: computed((): string => {
+        return props.defaultCategoryMessage
+      }),
+      searchTypeLabel: computed((): string => {
+        if (localState.selectedSearchType?.searchTypeUI) {
+          return localState.selectedSearchType.searchTypeUI
+        }
+        return 'Select a search category'
+      }),
+      origItems: computed((): Array<SearchTypeIF> => {
+        const allSearchTypes = []
+        if (isRoleStaffReg.value) {
+          if (getFeatureFlag('bcregistry-ui-ppr-mhr-staff-only')) {
+            allSearchTypes.push.apply(allSearchTypes, SearchTypes)
+            allSearchTypes.push.apply(allSearchTypes, MHRSearchTypes)
+            return allSearchTypes
+          } else {
+            return SearchTypes
+          }
+        }
+        if (hasPprRole.value) {
+          allSearchTypes.push.apply(allSearchTypes, SearchTypes)
+          // we can pop the title off if there is only one search type
+          if (!hasMhrRole.value) {
+            allSearchTypes.shift()
+          }
+        }
+        if (hasMhrRole.value) {
```

Comment:
same thing here, you can use the same feature flag instead of the role so that we can turn it off